### ISSUE TITLE
v0.5.0 Causes breaking change for certain model outputs.

### ISSUE THREAD

**Issue by dchoi-specificmedia** [viant/mly#19]
For further non-breaking changes, please clean up the `examples` directory and ensure `e2e` is working.

**Comment by dchoi-specificmedia** [comment]
```python
class LookupLayer(layers.Layer):
    def __init__(self, keys, values, default_value, name=None, table_name=None):
        self.keys = keys
        self.values = values
        self.default_value = default_value
        self.table_name = table_name
        super().__init__(name=name, trainable=False)

    def build(self, input_shape):
        keys_tensor = tf.constant(self.keys)
        values_tensor = tf.constant(self.values)
        kv_init = tf.lookup.KeyValueTensorInitializer(keys=keys_tensor, values=values_tensor)
        table = tf.lookup.StaticHashTable(kv_init, default_value=self.default_value, name=self.table_name)
        self.table = table

    def call(self, inputs):
        return self.table.lookup(inputs)

if __name__ == '__main__':
    # sample.csv can have columns s1, s2, i1, i2 for testing
    lookup_source = pd.read_csv('sample.csv').set_index('label')
    str_cols = ['s1', 's2']

    str_lookup_layers = [LookupLayer(keys, lookup_source[c].to_numpy(dtype='str'), default_value='', name=c) for c in str_cols]
    int_lookup_layers = [LookupLayer(keys, lookup_source[c].to_numpy(), default_value='', name=c) for c in int_cols]
    lookup_layers = str_lookup_layers + int_lookup_layers

    outputs = [lookup_layer(graph) for lookup_layer in lookup_layers]
    wrapped_model = models.Model(inputs=[lat, lon], outputs=outputs, name='broken_case_v0_5_0')
```

**Comment by dchoi-specificmedia** [comment]
While digging through this code, there are two other issues to address:

1. Dictionary hashes for models with wildcards may not be working as expected - changes to model dictionary may not be invalidating caches as appropriate.
2. The local cache on the server side is no longer relevant - in commit 3f8d43f87a96f6fc203bb3aa39ad5fad6c9be9b4 the read from local cache was removed, and now there's only a write to local and L1 backplane.

**Comment by dchoi-specificmedia** [comment]
Should be addressed via v0.6.0 tag.
